spec/integration_spec.rb

```diff
@@ -10,23 +10,39 @@
 Thread.abort_on_exception = true
 describe "A client" do
   let(:certificate) { Flores::PKI.generate }
-  let(:certificate_file_crt) { "certificate.crt" }
-  let(:certificate_file_key) { "certificate.key" }
+  let(:certificate_file_crt) do
```

Comment:
Yes, I have created `let_tmp_file` after that spec file, it was pissing me off to repeat the code :P

but yes will replace all theses by it, since it makes the code more readable.